conda_forge_tick/pypi_name_mapping.py

```diff
@@ -154,6 +156,8 @@ def extract_pypi_information(cf_graph: str) -> List[Dict[str, str]]:
     # TODO: exclude archived node_attrs
     for f in list(glob.glob(f"{cf_graph}/node_attrs/*.json")):
         meta_yaml = load_node_meta_yaml(f)
+        if meta_yaml is None:
```

Comment:
I think the next clause covers this

Reply:
it does, but this keeps mypy completely happy